scripts/zones/Phomiuna_Aqueducts/npcs/_0rm.lua

```diff
@@ -1,10 +1,8 @@
 -----------------------------------
--- Area: Phomiuna_Aqueducts
---  NPC: _0rm (Oil lamp)
--- Notes: Opens South door at J-7 from inside.
--- !pos -63.703 -26.227 83.000 27
------------------------------------
-require("scripts/globals/missions");
+--    Area: Phomiuna_Aqueducts
+--  NPC: Oil Lamp - North (West)
+-- Opens Door at F-7 from inside.
+--  ID: 16888052 !pos -63 -26 83
```

Comment:
NPC IDs can/do shift, so these comments might fall out of date and confuse people later.

Reply:
IDs deleted.